CHANGELOG.md

```diff
@@ -4,6 +4,28 @@ All notable changes to this project will be documented in this file.
 Each new release typically also includes the latest modulesync defaults.
 These should not affect the functionality of the module.
 
+## [v10.0.0](https://github.com/voxpupuli/puppet-redis/tree/v10.0.0) (2024-02-16)
+
+[Full Changelog](https://github.com/voxpupuli/puppet-redis/compare/v9.3.0...v10.0.0)
+
+**Breaking changes:**
+
+- remove legacy Debian 10 / Ubuntu 18.04 / CentOS 7 [\#504](https://github.com/voxpupuli/puppet-redis/pull/504) ([rwaffen](https://github.com/rwaffen))
+
+**Implemented enhancements:**
+
+- Add Ubuntu 22.04 [\#505](https://github.com/voxpupuli/puppet-redis/pull/505) ([rwaffen](https://github.com/rwaffen))
+- Implement rudimentary Redis and Sentinel ACLs [\#502](https://github.com/voxpupuli/puppet-redis/pull/502) ([TwizzyDizzy](https://github.com/TwizzyDizzy))
+
+**Closed issues:**
+
+- Error when deploying a redis instance [\#500](https://github.com/voxpupuli/puppet-redis/issues/500)
+- ACL support is missing [\#446](https://github.com/voxpupuli/puppet-redis/issues/446)
```

Comment:
I added skip-changelog here because we've the related PR already in the changelog